toml/src/main/kotlin/org/rust/toml/inspections/CrateNotFoundInspection.kt

```diff
@@ -55,9 +56,11 @@ class CrateNotFoundInspection : TomlLocalInspectionToolBase() {
             if (property in dependency.properties) return
         }
 
-        if (CratesLocalIndexService.getInstance().getCrate(dependency.crateName) == null) {
-            holder.registerProblem(dependency.crateNameElement, "Crate ${dependency.crateName} not found")
-        }
+        try {
+            if (CratesLocalIndexService.getInstance().getCrate(dependency.crateName) == null) {
```

Comment:
BTW I notice that you use two different styles of accessing to the index:
```Kotlin
val cratesService = CratesLocalIndexService.getInstance()
cratesService.getCrate(crateName)
```
and
```Kotlin
CratesLocalIndexService.getInstance().getCrate(crateName)
```

Please choose only one of them for the code style consistency. I'd recommend the second one since the instance variable is accessed only once.